ocfweb/docs/docs/services/web/django.md

```diff
@@ -1,91 +1,97 @@
+   well. This part varies by how you're organizing your project.
 
-1.   Change the permissions of the settings.py file, since it might have
-     sensitive information like your database password!
+4. Make a directory under `public_html` to house your application. For example,
+   `~/public_html/django`. You can also just use `public_html` directly if
+   desired.
 
-         tsunami$ chmod go-rwx settings.py
+5. In the directory you just created, make a file called `run.fcgi` with the file contents:
 
-1.   Create an app, edit it, and so on:
+   ```python
+   #!/home/u/us/user/app/venv/bin/python
+   import os
+   import sys
 
-         tsunami$ django-admin startapp miku
-         tsunami$ vim miku/views.py
-         ...
-         tsunami$ vim urls.py
+   sys.path.insert(0, os.path.expanduser('~/app'))
 
-1.    Make a directory in your ~/public_html directory:
+   from flup.server.fcgi import WSGIServer
+   from djangoapp import wsgi
 
-        tsunami$ mkdir ~/public_html/hatsune
+   if __name__ == '__main__':
+       WSGIServer(wsgi.application).run()
+   ```
 
-1.    Go into the directory you just created, and create two files: *.htaccess*
-      (note the dot), and *run.fcgi*.
+   Make sure to replace the first line of `run.fcgi` file with the actual path
+   to your project's virtualenv Python. You can find it by running the command
+   `readlink -f ~/app/venv/bin/python`.
 
-* .htaccess:
+   Make sure also to replace `from djangoapp` with the name of your Django
+   application (that's the name of the directory containing your `wsgi.py`
+   file).
 
-      RewriteEngine On
-      RewriteBase /
-      RewriteCond %{REQUEST_FILENAME} !-f
-      # Change "username" and "hatsune" to your username and whatever directory name you made in public_html, respectively
-      RewriteRule ^(.*)$ /~username/hatsune/run.fcgi/$1 [QSA,L]
+6. In the same directory, run `chmod +x run.fcgi`.
 
-* run.fcgi:
+7. In the same directory, create another file called `.htaccess` with these contents:
 
-      #!/usr/bin/env python
-      import sys, os
+   ```htaccess
+   RewriteEngine on
+   RewriteBase /
+   RewriteCond %{REQUEST_FILENAME} !-f
+   # Change "user" and "django" to your username and whatever directory
+   # name you made in public_html, respectively.
+   RewriteRule ^(.*)$ /~user/django/run.fcgi/$1 [QSA,L]
+   ```
 
-      # Change this path to reflect the real directory where your django-project lives
-      projects_location = "/home/k/ke/kedo/django"
-      project_name = "hatsune"
+Your app should now be accessible!
 
-      sys.path.insert(0, projects_location)
-      sys.path.insert(1, os.path.join(projects_location, project_name))
-      os.chdir(projects_location)
+Once your app has started running, changes you make to the Python code or
+templates won't take effect for a few hours. To apply changes immediately, you
+can touch the `run.fcgi` file with the command:
```

Comment:
Maybe say "make the `run.fcgi` file more recent" or "tell the webserver to reload your app" instead of "touch" since it's slightly esoteric terminology.